website/content/docs/platform/k8s/vso/index.mdx

```diff
@@ -168,6 +170,11 @@ Provides the configuration necessary for the Operator to synchronize a single Va
 Supported secrets engines *non-exhaustive*: [databases](/vault/docs/secrets/databases), [aws](/vault/docs/secrets/aws),
 [azure](/vault/docs/secrets/azure), [gcp](/vault/docs/secrets/gcp), ...
 
+##### Database secret example
+
+In this example, the database secret engine is mounted at `db` path. A Vault role named `my-postgresql-role`
+is configured to map a name in Vault to an executable database statement to create the database credential.
```

Comment:
I love the concise detail here, but I wonder we should link out to the related database secret's engine docs instead. Also we probably want to re-mention that `path` is under `mount` per my comments above.